themes/src/main/resources-community/theme/base/login/messages/messages_ja.properties

```diff
@@ -321,3 +339,33 @@ openshift.scope.list-projects=プロジェクトの一覧表示
 saml.post-form.title=認証リダイレクト
 saml.post-form.message=リダイレクトしています。お待ちください。
 saml.post-form.js-disabled=JavaScriptが無効になっています。有効にすることを強くお勧めします。継続するには、下のボタンをクリックしてください。
+
+#authenticators
+otp-display-name=オーセンティケーター・アプリケーション
+otp-help-text=オーセンティケーター・アプリケーションから取得した確認コードを入力してください。
+password-display-name=パスワード
+password-help-text=パスワードを入力してログインします。
+auth-username-form-display-name=ユーザー名
+auth-username-form-help-text=ユーザー名を入力してログインを開始します
+auth-username-password-form-display-name=ユーザー名とパスワード
+auth-username-password-form-help-text=ユーザー名とパスワードを入力してログインしてください。
+
+# WebAuthn
+webauthn-display-name=セキュリティー・キー
```

Comment:
I know this conforms JTF style guide, but "セキュリティー・キー" sounds weird for me. 
I prefer "セキュリティーキー", because most documents in the internet is written in this style, and JTF style guide also says "語句が短い場合には区切りを入れない方が望ましい。".

Reply:
う、ここも一括置換ミスりました...
- セキュリティーキー
- セキュリティーキー・ログイン

ですね

Reply:
I confirmed the fix.